.github/workflows/release.yml

```
@@ -0,0 +1,78 @@
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: simple
+          extra-files: |
+            ./VERSION
+
+      - name: Checkout
+        if: ${{ steps.release.outputs.release_created }}
+        uses: actions/checkout@v2
+
+      - name: Install Dependencies
+        if: ${{ steps.release.outputs.release_created }}
+        env:
+          CMAKE_VERSION: 3.27
+          CMAKE_BUILD: 0
```

we can change this to
```suggestion
          CMAKE_VERSION: ${{vars.CMAKE_VERSION}}
          CMAKE_BUILD: ${{vars.CMAKE_BUILD}}
```
so that these values can be controlled from your repos settings panel instead of requiring a PR to change.